components/common/AccountTag/index.tsx

```diff
@@ -0,0 +1,63 @@
+import classNames from 'classnames';
+
+type TagTypes = {
```

Comment:
We can move this to types file, if we maybe need to use it somewhere else as well, if not, it can stay here

Reply:
no need. it isnt used anywhere but this file